libpyvinyl/Parameters/Collections.py

```diff
@@ -24,8 +27,7 @@ def check_type(self, parameter):
         Checks given parameter is of type Parameter
         """
         if not isinstance(parameter, Parameter):
-            raise RuntimeError(
-                "A non-Parameter object was given to Parameters class.")
+            raise RuntimeError("A non-Parameter object was given to Parameters class.")
```

Comment:
Could make this even more explicit: `"Object of type Parameter expected, received {}".format(type(parameter))`

Reply:
That's a good suggestion.

Reply:
Ok